### PR TITLE
New Applicant Statistics

### PR DESCRIPTION
**Pull request by kenrhe**

### CODE REVIEW

File: application/mod_stats/controllers.py [hunk 1]
```diff
@@ -24,12 +24,27 @@ def get_application_stats():
...
 #Only for submitted
 def get_applicant_stats():
   stats = []
   num_submitted = UserEntry.objects(status = "Submitted").count()
+  num_accounts_submitted = UserEntry.objects(confirmed = True, status = "Submitted").count()
+
+  num_beginners = UserEntry.objects(confirmed=True, beginner="yes", status="Submitted").count()
+  num_males = UserEntry.objects(confirmed=True, gender="male", status="Submitted").count()
+  num_females = UserEntry.objects(confirmed=True, gender="female", status="Submitted").count()
+  num_rathernotsay = UserEntry.objects(confirmed=True, gender="rns", status="Submitted").count()
```
Comment (JackNeus) [inline application/mod_stats/controllers.py]: This isn't accurate as we do not collect gender for mentors.

File: application/mod_stats/controllers.py [hunk 2]
```diff
@@ -24,12 +24,27 @@ def get_application_stats():
...
+
+  stats.append('%d%% beginners (%d)' % (100.0 * num_beginners / num_accounts_submitted, num_beginners))
+  stats.append('%d%% female (%d)' % (100.0 * num_females / num_accounts_submitted, num_females))
+  stats.append('%d%% male (%d)' % (100.0 * num_males / num_accounts_submitted, num_males))
+  stats.append('%d%% rather not say (%d)' % (100.0 * num_rathernotsay / num_accounts_submitted, num_rathernotsay))
```
Comment (JackNeus) [inline application/mod_stats/controllers.py]: Can you make separate variables to store the percentages? See get_application_stats().

File: application/mod_stats/controllers.py [hunk 3]
```diff
@@ -24,12 +24,27 @@ def get_application_stats():
   stats.append('%d%% Applications have status \"In Progress\" (%d)' % (percent_accounts_in_progress, num_accounts_in_progress))
   stats.append('%d%% Applications have status \"Submitted\" (%d)' % (percent_accounts_submitted, num_accounts_submitted))
 
+  num_mentors = UserEntry.objects(confirmed=True, type_account="mentor").count()
```
Comment (JackNeus) [inline application/mod_stats/controllers.py]: Please space '=' signs to maintain consistent code.

File: application/mod_stats/controllers.py (hunk 1 above)
Comment (kenrhe) [inline application/mod_stats/controllers.py]: If we don't collect gender for mentors, why would it have an affect on this statistic?

File: application/mod_stats/controllers.py (hunk 2 above)
Comment (kenrhe) [inline application/mod_stats/controllers.py]: Sure.

File: application/mod_stats/controllers.py (hunk 1 above)
Comment (JackNeus) [inline application/mod_stats/controllers.py]: Because you're dividing by num_submitted which includes mentor accounts.

**Comment by rkty13** [comment]
Each of these calculations should go in their own controller functions so that we don't have to reuse code if we ever need these stats separately.

**Comment by rkty13** [comment]
Also, properly capitalize your commit messages and add periods please. We're trying to be consistent.

**Comment by JackNeus** [comment]
@rkty13 Feel free to refactor everything

**Pull request merged**